1.6/Dockerfile.windows

```diff
@@ -0,0 +1,12 @@
+FROM windowsservercore
+
+ENV GOLANG_VERSION 1.6.1
+ENV GOLANG_DOWNLOAD_URL "https://golang.org/dl/go$GOLANG_VERSION.windows-amd64.zip"
+
+RUN powershell -Command \
+	(New-Object System.Net.WebClient).DownloadFile('%GOLANG_DOWNLOAD_URL%', 'go.zip') ; \
+	Expand-Archive go.zip -DestinationPath c:\\ ; \
+	Remove-Item go.zip -Force
+
+RUN setx /M PATH "C:\go\bin;%PATH%" && \
```

Comment:
My $.02 - I truly don't buy that in something so simple. But your call as to what goes into any official images. Just giving my opinion :smile: